docs_src/getting-started/random-generator-device.yaml

```diff
@@ -7,34 +7,18 @@ labels:
 description: "random number generator to simulate a device"
 
 deviceResources:
-    -   
-        name: "randomnumber"
-        description: "generated random number"
-        attributes:
-            { type: "random" }
-        properties:
-            value:
-                { type: "int32", readWrite: "R" }
-            units:
-                { type: "String", readWrite: "R" }
+  -
+    name: "RandomNumber"
+    description: "generated random number"
+    attributes:
+      { type: "random" }
+    properties:
+      valueType: "Int32"
+      readWrite: "R"
 
 deviceCommands:

```

Comment:
Remove the corresponding Device Command section, because it is just the same as the corresponding Device Resource